modules/ROOT/partials/configuration/tinycomments_delete_comment.adoc

```diff
@@ -54,6 +53,6 @@ tinymce.init({
   tinycomments_delete: delete_comment_thread, // Add the callback to TinyMCE
   tinycomments_delete_all: delete_all_comment_threads,
   tinycomments_delete_comment: delete_comment,
-  tinycomments_lookup: lookup_comment
+  tinycomments_lookup: fetch_comments,
```

Comment:
Why ? I expect there to be both the lookup and the fetch callbacks provided.
```suggestion
  tinycomments_lookup: lookup_comment,
  tinycomments_fetch: fetch_comments
```

Reply:
typo, thanks for picking this up.